### ISSUE TITLE
Commandline vs GUI

### ISSUE THREAD

**Issue by github-userx** [caltrop2075/FAT-Catalog#1]
Hi Liz!

when/before you wrote this, did you also check out some of the GUi tools on Linux for cataloging usb/hdd drives and DVDs etc? I found some new promising tools on GitHub, mostly GUI apps written by single developers. Because the old tools like vvv, GuView etc have mostly been abandoned for years and often don’t even work on modern distros anymore.

**Comment by caltrop2075** [comment]
nah, no GUI wanted, this does what I want and is easily modified
